Demo/UITests/BraintreeDropIn_UITests.swift

```diff
@@ -303,9 +303,9 @@ class BraintreeDropIn_CardholderNameAvailable_UITests: XCTestCase {
 
         app.buttons["Add Card"].forceTapElement()
 
-        waitForElementToAppear(app.staticTexts["ending in 11"])
+        waitForElementToAppear(app.staticTexts["••• ••11"])
 
-        XCTAssertTrue(app.staticTexts["ending in 11"].exists)
+        XCTAssertTrue(app.staticTexts["••• ••11"].exists)
```

Comment:
So for cards, we currently use "••• ••11" in Drop-in. These tests are only changing because the demo app used "ending in 11" rather than the dots. But the Drop-in UI itself was not using "ending in 11".

We can look into how this would sound with VoiceOver. But we did verify that Android also uses the dots for cards.

Reply:
In general though - this a good callout for us to make sure we get translations for "ending in" to improve our screen reader experience. Currently, the screen reader will read "bullet bullet one one" on our vault screen for "••11".

I added a TODO for this in our SharePoint document for accessibility updates to iOS.

Reply:
I'm confused. Isn't having the screen reader say "bullet bullet one one" correct? Why would it say "ending in 11" if that's not what is shown on the screen?

Reply:
The screen reader currently reads "bullet bullet ... one one"

Reply:
That is a good point, because "bullet bullet one one" _is_ what the screen says. My only fear is that this might typically denote a bulleted list of items. I wonder what the protocol here is. We can reach out to the accessibility team too to get their thoughts.